api2-war/src/main/java/eu/europeana/api2/v2/service/translate/SearchResultTranslateService.java

```diff
@@ -32,10 +32,10 @@ public class SearchResultTranslateService {
 
     private final TranslationService translationService;
 
-    @Value("#{europeanaProperties['translation.truncate.after']}")
-    private Integer truncateFieldAfter;
-    @Value("#{europeanaProperties['translation.truncate.hardlimit']}")
-    private Integer truncateFieldHardLimit;
+    @Value("#{europeanaProperties['translation.char.Limit']}")
```

Comment:
It's a tiny thing, but to be consistent it's best to use all lowercase, so `translation.char.limit`

Reply:
done